application/python/example-driver.py

```diff
@@ -23,74 +23,36 @@
 import time
 
 # driver = DobotDriver('COM4')
-driver = DobotDriver('/dev/tty.usbmodem1421')
+driver = DobotDriver('COM4')
```

Comment:
Please retain the original com port as an example for windows already exists and is commented out above.